apps/docs/typedoc.mjs

```diff
@@ -27,7 +29,12 @@ export default {
   pageTitleTemplates: {
     member: (args) => args.name, // simpler page titles for member pages
   },
-  plugin: ['typedoc-plugin-markdown', './lib/typedoc-plugin-hypergiant.mjs'],
+  plugin: [
+    'typedoc-plugin-markdown',
+    './lib/typedoc-plugin-hypergiant.mjs',
+    './lib/playground-plugin-hypergiant.mjs',
```

Comment:
i kind of like the separation of concerns, but i'm open to including this in the general typedoc-plugin-hypergiant if that's what folks prefer

Reply:
Is there a reason you're adding the `-hypergiant` suffix?

Reply:
not really - just copying prior art. happy to change it if you prefer

Reply:
For this I would suggest `typedoc-plugin-playground.mjs` instead. I made the first one with the thought of it being filled with very small - possibly one-liners - of customization that would only really make sense for us. This one might be something we could generalize to be valuable to the community.

Reply:
updated here: https://github.com/gohypergiant/standard-toolkit/pull/171/commits/83b38b33cd259c3fc4d02daf007ea9fb02239a53